packages/mds-policy/api.ts

```diff
@@ -408,7 +403,7 @@ function api(app: express.Express): express.Express {
   /* eslint-enable @typescript-eslint/no-unused-vars */
 
   app.get(pathsFor('/test/initialize'), (req, res) => {
-    // FIXME janky
+    // TODO optimize
```

Comment:
How's this non-optimal?

Reply:
Could be simply:
```
try {
  const kind = await db.initialize()
  res.status(200).send({ result: '....' })
}
catch (err) {
  await log.error('initialize exception', ex)
  res.status(500).send(SERVER_ERROR)
}
```
}

Reply:
I'm really not sure. @karcass could you weigh in?

Reply:
I don't recall why I wrote that.  It's possible that the code got cleaned up but the comment was not removed.  I don't see any reason to keep the comment.